bind/gencffi.go

```diff
@@ -169,6 +196,24 @@ func (g *cffiGen) genCffiCdef() {
 func (g *cffiGen) genWrappedPy() {
 	n := g.pkg.pkg.Name()
 	g.wrapper.Printf(cffiHelperPreamble, n)
+	g.wrapper.Indent()
+	for _, s := range g.pkg.structs {
+		g.genStructConversion(s)
+	}
+	g.wrapper.Outdent()
+
+	g.wrapper.Printf("# make sure Cgo is loaded and initialized\n")
```

Comment:
you may want to comment here why we have to put the loading of cgo here and not in the preamble.